lib/recipes/bake

```diff
@@ -62,14 +64,19 @@ book_recipe = \
   when 'sociology' then [SOCIOLOGY_RECIPE]
   when 'statistics' then [STATISTICS_RECIPE]
   when 'u-physics' then [U_PHYSICS_RECIPE]
+  when 'web-generic' then [WEB_RECIPE]
   when 'world-history' then [WORLD_HISTORY_RECIPE]
   else
     raise 'Invalid recipe keyword'
   end
 
+recipe_array = book_recipes
+recipe_array.push(WEB_RECIPE) if opts[:platform] == 'web'
+recipe_array.push(VALIDATE_OUTPUT)
+
 puts Kitchen::Oven.bake(
   input_file: opts[:input],
-  recipes: [book_recipe, VALIDATE_OUTPUT].flatten,
+  recipes: recipe_array,
```

Comment:
Okay! So it bakes the corresponding recipe, the web recipe, and validates output every time it bakes then?